cockroachdb/templates/job.init.yaml

```diff
@@ -49,7 +49,7 @@ spec:
     {{- end }}
     {{- end }}
       restartPolicy: OnFailure
-      terminationGracePeriodSeconds: 300
+      terminationGracePeriodSeconds: {{ .Values.init.terminationGracePeriodSeconds }}
```

Comment:
If we are adding this, I would add it to the readme

Reply:
Done